core/io/file_access.h

```diff
@@ -143,7 +159,10 @@ class FileAccess : public RefCounted {
 	virtual void store_pascal_string(const String &p_string);
 	virtual String get_pascal_string();
 
-	virtual void store_buffer(const uint8_t *p_src, uint64_t p_length); ///< store an array of bytes
+	virtual void _store_buffer(const uint8_t *p_src, uint64_t p_length); ///< store an array of bytes
```

Comment:
When this happens, we generally do the opposite: The underscore version is private and the one that goes to the bindings (that you can re-bind without underscore in the binding name), and C++ remains using the regular one.

This way, you don't need to change any code.

Reply:
as reference of how its done in other places:
https://github.com/godotengine/godot/blob/master/servers/rendering/rendering_device.h#L1300

Reply:
I did it based on this comment: https://github.com/godotengine/godot/pull/65271#discussion_r962123140

Reply:
Since there is only one place using in the `TextServer`, and it's only an issue for building the same code as module and GDExtension, I guess I can add a `_*_buffer` wrapper methods directly to the `godot-cpp`, or just use `ifdef` in the `TextServer` for this one case.

Currently, `TextServer` is not buildable as GDExtension, due to lack of `TypedArray`s (working on it https://github.com/godotengine/godot-cpp/pull/841), and it's also using inconsistent virtual methods without underscore, so it's gonna need some extra work anyway.